dm/dm/master/scheduler/scheduler.go

```diff
@@ -1194,7 +1185,27 @@ func (s *Scheduler) GetSubTaskCfgs() map[string]map[string]config.SubTaskConfig
 		clone[task] = clone2
 		return true
 	})
+	return clone
+}
 
+// GetSubTaskCfgs gets all subTask config pointer, return nil when error happens.
+func (s *Scheduler) GetALlSubTaskCfgs() map[string]map[string]*config.SubTaskConfig {
```

Comment:
we need to sync the whole method, since the item stored in `s.subTaskCfgs` is a normal map which may changed concurrently

Reply:
`sheduler.subTaskCfgs` is a `sync.Map`

Reply:
it's a double map, the inside map is not a `sync.Map`